test/journeys/specs/space/global/messaging.js

```diff
@@ -19,6 +20,9 @@ import {
 describe('Widget Space', () => {
   const browserLocal = browser.select('browserLocal');
   const browserRemote = browser.select('browserRemote');
+  const jobName = 'react-widget-space-global';
```

Comment:
Almost feel like these job names should be in a shared file somewhere so we don't get strings mucked up along the way.

Reply:
Sure, done!